src/styles/minute.scss

```diff
@@ -0,0 +1,18 @@
+.container__minute {
+  position: absolute;
+  left: $firstPosition;
+  top: 225px;
```

Comment:
Add container__second and container__minute to container.scss file, it's elements of one block, you shouldn't create separate file for each element